pages.fr/common/reboot.md

```diff
@@ -0,0 +1,20 @@
+# reboot
+
+> Réinitialisez le système.
+> Plus d'informations : <https://manned.org/shutdown.8>.
```

Comment:
```suggestion
> Plus d'informations : <https://www.man7.org/linux/man-pages/man8/reboot.8.html>.
```

As I understand you are making translation for [this](https://github.com/tldr-pages/tldr/blob/main/pages/linux/reboot.md) page. It's right why it's in `common/` and not in `linux/`?